web-frontend/src/layouts/Placeholder/Placeholder.tsx

```diff
@@ -0,0 +1,23 @@
+import { FiList } from 'react-icons/fi';
+import Intl from '../../i18n/i18n';
+
+function Placeholder() {
+  return (
+    <div
+      className="flex flex-col justify-around items-center  
+        h-[350px] w-[680px] p-10 gap-10
```

Comment:
Don't we have some correspondence in tailwind to avoid custom sizes!?

Reply:
only goes up to 96 so I added that to the h